plugins/python-build/bin/python-build

```diff
@@ -1508,7 +1508,8 @@ use_macports() {
   if [ -n "$port_location" ]; then
     local prefix="${port_location%/bin/port}"
     export CPPFLAGS="${CPPFLAGS:+$CPPFLAGS }-I${prefix}/include"
-    append_ldflags_libs "-L${prefix}/lib -Wl,-rpath,${prefix}/lib"
+    prepend_ldflags_libs "-L${prefix}/lib -Wl,-rpath,${prefix}/lib"
```

Comment:
I tried to get `configure`'s trace as you suggested and found an interesting behavior.

Appending the `-L/opt/local/lib -Wl,-rpath,/opt/local/lib` resulted in
```
*** WARNING: renaming "readline" since importing it failed: dlopen(/Users/danny/.pyenv/sources/3.11.13/Python-3.11.13/build/lib.macosx-15.5-arm64-3.11/readline.cpython-311-darwin.so, 0x0002): symbol not found in flat namespace '_rl_catch_signals'
```
while prepending the flag didn't have any issues. It's a little confusing because I see `use_homebrew()` appends the flags without issues.

Attaching the build logs with traces below:
[build-append.log](https://github.com/user-attachments/files/20732338/build-append.log)
[build-prepend.log](https://github.com/user-attachments/files/20732346/build-prepend.log)

Reply:
> Do you suggest prepending `CPPFLAGS` to match `LDFLAGS` order like this?

Yes

> Or is this an upstream issue in macports like the homebrew issue you linked.

What I linked is not an "upstream issue". It's just a thread I googled to find out what this error might mean -- which just happened to some Homebrew issue thread.